devtools/Pipfile

```diff
@@ -13,8 +12,13 @@ behave = "*"
 pytest = "*"
 rdflib = "*"
 python-dateutil = "*"
-pipenv-setup = "*"
 csvw = "*"
+sphinx = "*"
+sphinx-book-theme = "*"
+pipenv-setup = "*"
+chardet = "*"
+black = "*"
```

Comment:
Black should be in dev-packages.

Reply:
It is.

Reply:
But if we want it to be automatically installed in down-stream packages, it needs to be outside the dev section here.